.github/workflows/staticcheck.yaml

```diff
@@ -21,4 +22,7 @@ jobs:
       - id: run-test
         run: yarn test:ci
       - id: codecov-script
-        run: ./scripts/codecov.sh
```

Comment:
may we remove the shell script too?

Reply:
Good catch! I'll do it!

Reply:
Done in 71c4d3a9c72475f3e374d1511cd38cfc267c0499